libs/barista-components/overlay/src/overlay.ts

```diff
@@ -107,10 +111,14 @@ export class DtOverlay implements OnDestroy {
   constructor(
     private _injector: Injector,
     private _overlay: Overlay,
+    // @ts-ignore unused variable
     private _viewportRuler: ViewportRuler,
     // tslint:disable-next-line:no-any
+    // @ts-ignore unused variable
     @Inject(DOCUMENT) private _document: any,
+    // @ts-ignore unused variable
     private _platform: Platform,
+    // @ts-ignore unused variable
```

Comment:
If they are unused can we deprecate them?

Reply:
Plan is to deprecated the entire component. That's why I didn't add a deprecation notice per argument.